app/frontend/helpers/submitFeedbackToSentry.js

```diff
@@ -0,0 +1,30 @@
+const sleep = require('../helpers/sleep')
+const debugLog = require('./debugLog')
+const {ADALITE_CONFIG} = require('../config')
+
+async function sendFeedback(comments, email, name, eventId) {
+  await sleep(5000) // to ensure the feedback gets send after the error
+  if (!comments) return
+  const params = {
+    event_id: eventId,
+    name,
+    email,
+    comments,
+  }
+  const url = 'https://sentry.io/api/0/projects/vacuumlabs-sro/adalite-frontend/user-feedback/'
```

Comment:
check constants.js file, I'd place the URL there. It'd be easier to maintain.

Reply:
thanks for suggestion, makes sense..